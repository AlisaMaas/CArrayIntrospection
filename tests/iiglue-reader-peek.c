int peek(int *values, int slot)
{
  return values[slot];
}
