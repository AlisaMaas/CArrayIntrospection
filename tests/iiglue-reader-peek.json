{"libraryEnums":[],"libraryFunctions":[{"foreignFunctionLinkage":"LinkDefault","foreignFunctionParameters":[{"parameterType":{"contents":{"contents":32,"tag":"CInt"},"tag":"CPointer"},"parameterName":"values","parameterAnnotations":[{"contents":[],"tag":"PANotNull"},{"contents":1,"tag":"PAArray"}]},{"parameterType":{"contents":32,"tag":"CInt"},"parameterName":"slot","parameterAnnotations":[]}],"foreignFunctionReturnType":{"contents":32,"tag":"CInt"},"foreignFunctionName":"peek","foreignFunctionAnnotations":[]}],"libraryName":"iiglue-reader-peek","libraryTypes":[],"libraryDependencies":[],"libraryAnnotations":[]}