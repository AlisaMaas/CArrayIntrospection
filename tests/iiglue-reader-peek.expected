Printing analysis 'Read iiglue analysis results and add them as metadata on corresponding LLVM entities':
	array arguments:
		peek::values
